### PR TITLE
Fix tag-release step in workflow which prevented github releases from being created automatically

### LINKED ISSUE #2744: Step tag-release in workflow fails
https://github.com/Mailu/Mailu/actions/runs/4616081958/jobs/8160968171#step:7:15

The step were the release is created fails for unknown reason. This must be resolved asap.

### PR DESCRIPTION
**Pull request by Diman0**
## What does this PR do?
Fix tag-release step in workflow which prevented github releases from being created automatically.

Cause was that a specific method is required for assigning multi-line strings in github workflow files: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#multiline-strings

On my fork Mailu_Fork
Replicated the issue:
https://github.com/Diman0/Mailu_Fork/actions/runs/4626459175

Fix in action:
https://github.com/Diman0/Mailu_Fork/actions/runs/4626683488/jobs/8183697136
https://github.com/Diman0/Mailu_Fork/releases/tag/1.9.28

## What type of PR?

bug fix

### Related issue(s)
- closes #2744 

## Prerequisites
Before we can consider review and merge, please make sure the following list is done and checked.
If an entry in not applicable, you can check it or remove it from the list.

- [n/a ] In case of feature or enhancement: documentation updated accordingly
- [x] Unless it's docs or a minor change: add [changelog](https://mailu.io/master/contributors/workflow.html#changelog) entry file.

### CODE REVIEW

**Comment by nextgens** [comment]
Should we squeeze in the changes suggested by @ghostwheel42 with it since it's a backport?

**Comment by Diman0** [comment]
Hmm. That is not a bad idea. That also allows me to state in the upgrade section that you need to have a look at your rspamd override files as well (modify the path to /overrides for the location of your map files).

I will include his suggestions. This will take some time.

**Comment by Diman0** [comment]
bors try

**Comment by Diman0** [comment]
@nextgens Please read the updated instructions in the `update section` in releases.rst. I'm correct that you need to add autoconfig.* to `HOSTNAMES=` in mailu.env to generate a certificate for this, am I not?
This certificate is otherwise not generated as far as I know.

File: docs/releases.rst [hunk 1]
```diff
@@ -292,10 +306,19 @@ Upgrading
...
+
+If you use Fail2Ban, then the Fail2Ban intructions have been improved. It is **mandatory** to remove your Fail2Ban config
+and re-apply it using the instructions from :ref:`updated Fail2Ban documentation <Fail2Ban>`.
 
-If you use Fail2Ban, then the Fail2Ban intructions have been improved. It is mandatory to remove your Fail2Ban config and re-apply it using the instructions from :ref:`updated Fail2Ban documentation <Fail2Ban>`.
+If you use overrides for Rspamd, then please note that overrides are now placed in the location `/overrides` in the rspamd container.
+If you use your own map files, change the location to /override/myMapFile.map in the corresponding rspamd conf file.
 
 To use the new autoconfig endpoint and Mailu RESTFul API, you may need to update your reverse proxy config.
+If you use ``TLS_FLAVOR=letsencrypt``, add autoconfig.myhostname.com to the setting ``HOSTNAMES=`` in mailu.env to generate a certifficate for the autoconfig endpoint as well.
```
Comment (nextgens) [inline docs/releases.rst:319]: Maybe hint at mta-sts in the FAQ too?

**Comment by Diman0** [comment]
I made spelling error `/overrides` is not the same as `/override`

File: docs/releases.rst (hunk 1 above)
Comment (Diman0) [reply docs/releases.rst:319]: Yes. That is a good idea. I will submit yet another commit.

Comment (nextgens) [reply docs/releases.rst:319]: oh and s/certifficate/certificate/

Comment (Diman0) [reply docs/releases.rst]: Fixed the typo and I added a hint to have a look at mta-sts. I hope my of mta-sts in layman terms  is correct.

**Comment by Diman0** [comment]
bors try

**Review by ghostwheel42 — APPROVED** [review]
